### PR TITLE
server: return domain path in login exception error message

### PR DESCRIPTION
**Pull request by shwstppr**
### Description

Minor fix to not show domain's DB ID in the error message that gets displayed in the UI.

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [x] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [x] Trivial

### Screenshots (if appropriate):
Before
Screenshot from 2021-03-24 15-06-07

After
Screenshot from 2021-03-24 15-31-55

### CODE REVIEW

**Review by rhtyd — APPROVED** [review]

**Comment by shwstppr** [comment]
@blueorangutan ui

**Review by Pearl1594 — APPROVED** [review]
Looks good

File: server/src/main/java/com/cloud/api/ApiServer.java [hunk 1]
```diff
@@ -1130,7 +1130,7 @@ public ResponseObject loginUser(final HttpSession session, final String username
 
             return createLoginResponse(session);
         }
-        throw new CloudAuthenticationException("Failed to authenticate user " + username + " in domain " + domainId + "; please provide valid credentials");
+        throw new CloudAuthenticationException("Failed to authenticate user " + username + " in domain " + userDomain.getPath() + "; please provide valid credentials");
```
Comment (rhtyd) [inline server/src/main/java/com/cloud/api/ApiServer.java:1133]: minor nit - General convention would be to override the `toString()`.

**Comment by shwstppr** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
I'm not in favour of this, it would give extra information to a hacker.

**Review by DaanHoogland — CHANGES_REQUESTED** [review]
let's not do this

**Comment by shwstppr** [comment]
> let's not do this

Path is already there in domain field in the form and there are similar exception messages in the code

**Comment by shwstppr** [comment]
Closing this as not a major issue. Will revisit later.

**Pull request closed without merging**